planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -1048,19 +1079,22 @@ std::optional<std::pair<PullOverPath, GoalCandidate>> GoalPlannerModule::selectP
       continue;
     }
 
-    const auto resampled_path = utils::resamplePathWithSpline(pull_over_path.getParkingPath(), 0.5);
+    // const auto resampled_path = utils::resamplePathWithSpline(pull_over_path.getParkingPath(),
+    // 0.5);
+    const PathWithLaneId parking_path = pull_over_path.getParkingPath();
```

Comment:
and bind by const reference &

Reply:
revert in [use copy getter](https://github.com/autowarefoundation/autoware.universe/pull/8195/commits/8ce336e0a1d1dddfb2ce7a10e82b97cab7685401)

becase the returned value is used in multi threads